packages/itwinui-react/package.json

```diff
@@ -127,7 +127,7 @@
     "react": "19.1.0",
     "react-dom": "19.1.0",
     "typescript": "5",
-    "vite": "~6.2.5",
+    "vite": "~6.2.7",
```

Comment:
Can we test `6.3.4` and update it if it works?

Reply:
It is now duplicated. Can all instances be updated to `6.3.4` or is 6.2.X being forced somewhere?